tests/test_enterprise/api/test_views.py

```diff
@@ -8663,22 +8608,6 @@ def test_successful_list_with_filters(self):
         assert len(enterprise_filtered_response.json().get('results')) == 1
         assert learner_filtered_response.json().get('results')[0].get('uuid') == str(new_group.uuid)
 
-    def test_list_members_little_bobby_tables(self):
-        """
-        Test that we properly sanitize member user query filters
-        https://xkcd.com/327/
-        """
-        # url: 'http://testserver/enterprise/api/v1/enterprise_group/<group uuid>/learners/'
-        url = settings.TEST_SERVER + reverse(
-            'enterprise-group-learners',
-            kwargs={'group_uuid': self.group_1.uuid},
-        )
-        # The problematic child
-        filter_query_param = "?user_query=Robert`); DROP TABLE enterprise_enterprisecustomeruser;--"
```

Comment:
also this bobby tables joke has to be in platform too

Reply:
yep! moving both tables to platform